### ISSUE TITLE
if user only is part of 1 attester org, automatically select it

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/DeVouch-FE#173]
If a user is only part of 1 attesterOrg it should be automatically selected, since it is the only option. 

![Image](https://github.com/user-attachments/assets/2a6143e4-939d-4d56-b675-acca43222b96)

**Comment by divine-comedian** [comment]
@mateodaza needs to work on it still

**Comment by divine-comedian** [comment]
@mateodaza what is the update on this issue?

**Comment by mateodaza** [comment]
yes @divine-comedian not sure why it didn't link the PR properly 
https://github.com/Giveth/DeVouch-FE/pull/182

they're live on staging

**Comment by LatifatAbdullahi** [comment]
@mateodaza

On staging, when Users with single attesters Org.  click to Vouch or Flag, the single Attester group is not selected by default.

Please take a look.


https://github.com/user-attachments/assets/e9f885d9-6015-4587-addd-254a7d7c3bb9

**Comment by mateodaza** [comment]
@LatifatAbdullahi thanks for that! this changed after we brought the non affiliation feature
this was fixed after this [PR](https://github.com/Giveth/DeVouch-FE/pull/190)

**Comment by LatifatAbdullahi** [comment]
Thanks @mateodaza it works as expected now.
